code.sh

```diff
@@ -9,25 +9,12 @@ if [ ! -f ${FIRST_RUN} ]; then
   touch ${FIRST_RUN}
 fi
 
-if [[ -d /usr/lib/sdk/dotnet ]]; then
-  . /usr/lib/sdk/dotnet/enable.sh
-fi
-
-if [[ -d /usr/lib/sdk/node10 ]]; then
-  . /usr/lib/sdk/node10/enable.sh
-fi
-
-if [[ -d /usr/lib/sdk/php73 ]]; then
-  . /usr/lib/sdk/php73/enable.sh
-fi
-
-if [[ -d /usr/lib/sdk/openjdk8 ]]; then
-  . /usr/lib/sdk/openjdk8/enable.sh
-fi
-
-if [[ -d /usr/lib/sdk/mono5 ]]; then
-  . /usr/lib/sdk/mono5/enable.sh
-fi
+IFS=',' read -ra SDK <<< "$FLATPAK_ENABLE_SDK"
+for i in "${SDK[@]}"; do
+  if [[ -d /usr/lib/sdk/$i ]]; then
+    . /usr/lib/sdk/$i/enable.sh
```

Comment:
Maybe add `/usr/lib/sdk/$i/bin` to `PATH` if `enable.sh` doesn't exist?

Reply:
Is there an extension that doesn't provide a `enable.sh`? I thought this is the standard?

Reply:
It's not mandatory. For example, `toolchain-i386` (built alongside with freedesktop-sdk itself) doesn't have it.

Reply:
```suggestion
    if [[ -f enable.sh ]]; then
      . /usr/lib/sdk/$i/enable.sh
    else
      export PATH=$PATH:/usr/lib/sdk/$i/bin
    fi
```